### PR TITLE
WIP: Updated Debian packaging

### PR DESCRIPTION
**Pull request by oerdnj**
WIP: This needs better commit message and better comment here.

### CODE REVIEW

File: distro/pkg/deb/copyright [hunk 1]
```diff
@@ -1,48 +1,107 @@
...
- to the license terms contained in, the Simplified BSD
- License set forth in Section 4.c of the IETF Trust's
- Legal Provisions Relating to IETF Documents
- (http://trustee.ietf.org/license-info).
- .
- This version of this YANG module is part of RFC 6536; see
- the RFC itself for full legal notices.
+Copyright: 2018-2021 CESNET, z.s.p.o.
+	   2018-2021 Deutsche Telekom AG.
+License: Apache-2.0
```
Comment (michalvasko) [inline distro/pkg/deb/copyright:8]: The license of the whole project was changed to BSD-3-Clause.

File: distro/pkg/deb/sysrepo-plugind.lintian-overrides [hunk 2]
```diff
@@ -0,0 +1 @@
+sysrepo-plugind: no-manual-page usr/bin/sysrepo-plugind
```
Comment (michalvasko) [inline distro/pkg/deb/sysrepo-plugind.lintian-overrides:1]: Why is this needed if the manual page is now included?

File: distro/pkg/deb/sysrepo-tools.lintian-overrides [hunk 3]
```diff
@@ -0,0 +1,2 @@
+sysrepo: no-manual-page usr/bin/sysrepocfg
+sysrepo: no-manual-page usr/bin/sysrepoctl
```
Comment (michalvasko) [inline distro/pkg/deb/sysrepo-tools.lintian-overrides:2]: The manual pages are included, what is the purpose of this then?

**Review by michalvasko — CHANGES_REQUESTED** [review]

**Comment by fbezdeka** [comment]
On a clean environment (sbuilder) I saw the following when trying to build debian packages:

```
| dh_install: warning: Cannot find (any matches for) "usr/lib/systemd/system/sysrepo-plugind.service" (tried in ., debian/tmp)
```

cmake was unable to locate the systemd unit directory (cmake variable SYSTEMD_UNIT_DIR) and failed to install the systemd service file.

I fixed that by adding `systemd` to the build dependencies.

**Comment by michalvasko** [comment]
Thanks but Debian scripts were not updated yet based on Fedora RPM review changes. We are hoping to finish RPMs soon and then the Debian scripts and packages will also be updated and officially submitted.

**Pull request closed without merging**